spec/lib/mcb/courses_editor_spec.rb

```diff
@@ -15,6 +15,9 @@ def run_editor(*input_cmds)
   let(:email) { 'user@education.gov.uk' }
   let(:provider) { create(:provider, provider_code: provider_code) }
   let(:accredited_body) { create(:provider, :accredited_body) }
+  let!(:mathematics) { create(:subject, subject_name: "Mathematics") }
```

Comment:
can we start seeding these into factory?

Reply:
yup good idea, see 86e8ef9